task-definition.json

```diff
@@ -0,0 +1,33 @@
+{
+  "family": "devnet-demo-client",
+  "taskRoleArn": "arn:aws:iam::348099934012:role/ecsTaskExecutionRole",
+  "executionRoleArn": "arn:aws:iam::348099934012:role/ecsTaskExecutionRole",
+  "containerDefinitions": [
+    {
+      "name": "prototype-client",
```

Comment:
- Could we later on name the container like the ap? Or is it good practice to name them differently?
- (Unrelated question, but I'm using the opportunity :) )Where to see where the app is deployed?

Reply:
I tried to rename it in this PR, but it didn't work, because on the deployment the server is searching for `prototype-client`. I would have to re-setup the server to have a name change